service/keys/keys.go

```diff
@@ -12,6 +16,8 @@ type Keys interface {
 	GetNostrPublicKey() string
 	// Wallet Service Nostr secret key
 	GetNostrSecretKey() string
+	// GetBIP32ChildKey derives a BIP32 child key from the nostrSecretKey given a child key index
+	GetBIP32ChildKey(childIndex uint32) (string, error)
```

Comment:
Regarding naming, do we need to know it's BIP32? could it be `GetAppWalletKey(appId uint32)`?